uint32 auction_round
uint32 bidder_id
uint32 task_id
float64 bid
float64 task_end